### ISSUE TITLE
declaration-block-no-ignored-properties should allow float: none

### ISSUE THREAD

**Issue by octoxan** [stylelint/stylelint#1569]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

This rule should not throw an error if the float property is set to 'none'. You CAN float: none; and display: inline. I know you could put the float: left in a different media query, but that doesn't change the fact that float none is perfectly functional with display: inline. 

> Which rule, if any, is this issue related to?

declaration-block-no-ignored-properties

> What CSS is needed to reproduce this issue?

```css
@media (min-width: 600px) {
  .foo {
    float: left;
  }
}
@media (min-width: 900px) {
  .foo {
    float: none;
    display: inline;
  }
}
```

**Comment by m-allanson** [comment]
This may be related to #1238

**Comment by octoxan** [comment]
Ah I didn't see that one, very similar, but not completely the same. I'm thinking either "float: none;" should be ignored by this rule entirely, or the rule should have the option to give it things to ignore.

**Comment by jeddy3** [comment]
@octoxan Thanks for the writing up the issue.

Needing finer-grain checks within `declaration-block-no-ignored-properties` [is discussed](https://github.com/stylelint/stylelint/issues/1238#issuecomment-219236893) within #1238. This issue is now referenced from that one. So, I'm going to close this one. Feel free to track the other issue, and/or help us fix other issue in the `v7` branch, as it's one of only handful of issues left in the [`7.0.0` milestone](https://github.com/stylelint/stylelint/milestones/7.0.0).
